code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -148,17 +153,14 @@
 			R.stun(20)
 			R.reveal(100)
 			R.adjustHealth(50)
-		addtimer(CALLBACK(src, .proc/divine_explosion, round(created_volume/48,1),get_turf(holder.my_atom)), 2 SECONDS)
+		for(var/mob/living/carbon/C in get_hearers_in_view(effective_size,T))
+			if(iscultist(C))
+				to_chat(C, "<span class='userdanger'>The divine explosion sears you!</span>")
+				C.Paralyze(40)
+				C.adjust_fire_stacks(5)
+				C.IgniteMob()
 	..()
```

Comment:
so the explosion will happen at the beginning of the hallelujah, not at the end of it?

isn't the point of stunning the cultists supposed to be to keep them in place for the explosion?

Reply:
..() is the explosion which happens after the stun

Reply:
yes, but there's no delay for the explosion now like there was before

Reply:
no, before the stun to cultist was delayed, the explosion was instant